packages/block-editor/src/components/block-toolbar/style.scss

```diff
@@ -163,10 +163,21 @@
 		}
 	}
 
-	.block-editor-block-mover:not(.is-horizontal) .block-editor-block-mover__move-button-container {
+	.block-editor-block-mover .block-editor-block-mover__move-button-container {
 		width: auto;
 	}
 
+	.block-editor-block-mover.is-horizontal .block-editor-block-mover__move-button-container {
+		padding-left: 6px;
+		padding-right: 6px;
+
+		> * {
+			// Override !important padding rules set in block mover styles.
+			padding-left: 6px !important;
+			padding-right: 6px !important;
```

Comment:
I'm open to better options than using `!important` here but I couldn't see a means of avoiding the original `!important` rules set in the block mover component, based on a possible CSS class further up the DOM tree.

Reply:
Do you know why we're using `!important` here in the first place? I removed all of those declarations in the Inspector and didn't notice any issues 🤔

Reply:
These particular rules were introduced in https://github.com/WordPress/gutenberg/pull/40379 but might have only been a reworking of existing rules that used important.

There's a lot going on in that PR style wise so it might be easier to remove the important flags and test further if they really are required. Without having much prior context here I assumed they were added for a reason I hadn't quite put my finger on yet.

Reply:
It appears that in previous refactorings of these styles we lost some inline comments advising when the `! important` rules could be removed.

I've tweaked the styles directly related to the block mover buttons in https://github.com/WordPress/gutenberg/pull/49747/commits/04d4e311105d15897b13489608a0b3dd275dc204. 

Tomorrow, I'll try and remove the `!important` uses for the block drag handle as well and then double check the spacing still matches trunk (text label left/right buttons excepted).